tests/unit/test_req_file.py

```diff
@@ -7,7 +7,7 @@
 from mock import Mock, patch
 from pretend import stub
 
-import pip._internal.index
+import pip._internal.req.req_file  # this will be monkeypatched
```

Comment:
nitpick: I would put the comment above, since refactoring tools seem to handle that better vs trailing comments.

Reply:
Well, in my experience, they don't handle trailing comments and parenthesis well. It just so happens a lot of our imports have trailing comments *and* parenthesis. I could be wrong here, but that's my judgement.

Anyway, this one's pretty unambiguous and I'm not sure if moving it above is as good.